src/components/Clock.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+function formatedTime(time: Date) {
```

Comment:
The function name must begin with a verb and at the entrance you accept the whole date and not only the time.
```suggestion
function getTimeFromDate(date: Date) {
```